robot_controll/main.cpp

```diff
@@ -293,24 +297,23 @@ int main(int argc, char** argv) {
     // sm.loc->readMap( (char *)"../maps/botanicka.osm" );
     // sm.loc->readMap( (char *)"../maps/borsky2.osm" );
     // sm.loc->readMap( (char *)"../maps/pisek.osm" );
-    const std::string map_file = config["map"].as<std::string>();
-    sm.loc->readMap((char*)map_file.c_str());
+    const std::string map_file = Config::getString("map");
+    sm.loc->readMap((char*) map_file.c_str());
 
-    const double longitude = config["longitude"].as<double>();
-    const double latitude = config["latitude"].as<double>();
-    printf("%f %f\n", longitude, latitude);
+    const double longitude = Config::getDouble("longitude");
+    const double latitude = Config::getDouble("latitude");
 
-    sm.coor->running_mean_weight = config["running_mean_weight"].as<double>();
-    sm.coor->speed_down_dst = config["speed_down_dst"].as<double>();
+    sm.coor->running_mean_weight = Config::getDouble("running_mean_weight");
+    sm.coor->speed_down_dst = Config::getDouble("speed_down_dst");
 
     Ll point;
     point.longitude = longitude;
     point.latitude = latitude;
     sm.loc->setDestination(point);
     //sm.loc->readDestination((char*) "../destination.txt");
 
-    const std::string neural_net = config["neural_net"].as<std::string>();
-    sm.nn->load((char*)neural_net.c_str());
+    const std::string neural_net = Config::getString("neural_net");
```

Comment:
It is changed in https://github.com/Robotics-DAI-FMFI-UK/smely-zajko/pull/9/commits/d08b08f1d8466b1b0a247b11f16b6ff54a7673bb